python/tests/kernel/test_kernel_features.py

```diff
@@ -988,3 +990,228 @@ def kernel(argument: float):
 
     with pytest.raises(RuntimeError) as error:
         print(cudaq.draw(kernel))
+
+
+def test_user_error_op_attr_1446():
+
+    @cudaq.kernel
+    def test_kernel(nQubits: int):
+        qubits = cudaq.qvector(nQubits)
+        x(qubits)
+        # behavior is strange, applies h to 6 qubits now
```

Comment:
We should probably remove this comment since this PR presumably fixes the issue described in the comment. (Same for lines 1014, 1027, and 1040.)

Reply:
I agree, will remove